canopen/pdo/base.py

```diff
@@ -113,9 +112,9 @@ def export(self, filename):
                                           unit=var.od.unit)
                 for value, desc in var.od.value_descriptions.items():
                     signal.addValues(value, desc)
-                frame.addSignal(signal)
-            frame.calcDLC()
-            db.frames.addFrame(frame)
+                frame.add_signal(signal)
+            frame.calc_dlc()
+            db.frames.append(frame)
```

Comment:
I don't see a `frames` attribute documented in the `CanMatrix` API reference.  It might work, but there seem to be several `add_frame_*()` methods which look more official.

Would you please check if any of those would be suitable here?